src/RedisSessionStateProvider/RedisSessionStateProvider.cs

```diff
@@ -162,15 +162,12 @@ public override async Task CreateUninitializedItemAsync(HttpContextBase context,
         {
             try
             {
-                if (LastException == null)
-                {
-                    LogUtility.LogInfo("CreateUninitializedItem => Session Id: {0}, Session provider object: {1}.", id, this.GetHashCode());
```

Comment:
I believe we still need the contents of these `if` blocks, but remove the `if` condition that prevents running them if `LastException != null`

Reply:
@stanleysmall-microsoft Right you are - I misread the diff